caikit/runtime/grpc_server.py

```diff
@@ -120,14 +120,14 @@ def __init__(
             )
 
             # Add training management servicer to the gRPC server
-            training_management_service: ServicePackage = (
+            self.training_management_service: ServicePackage = (
```

Comment:
I added this to enable referencing the `training_management_service` off of an in-memory server handle. This is very likely not a production usage, but it mirrors how `self.training_service` and `self.inference_service` are stored.